lms/djangoapps/course_home_api/dates/v1/views.py

```diff
@@ -52,10 +55,15 @@ class DatesTabView(RetrieveAPIView):
     **Returns**
 
         * 200 on success with above fields.
-        * 403 if the user is not authenticated.
+        * 401 if the user is not authenticated.
         * 404 if the course is not available or cannot be seen.
     """
 
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
+    )
```

Comment:
When looking at where `BearerAuthenticationAllowInactiveUser` is used, _almost_ all of the APIs included these two other authentication classes. The `SessionAuthenticationAllowInactiveUser` is what allowed the tests to pass as written so I'm pretty confident we want that one. I believe the `JwtAuthentication` is just to support that flow, but I'm not sure it is currently used.

Reply:
So presumably the default was Jwt and Session?

Reply:
The default definitely wasn't JWT and Session (or at the least, not in that order). Elsewhere in this PR, I updated the response code for unauthenticated/no permission from 403 to 401. The reason for this change is that the first listed Auth Class is what determines the return code. Since putting JWT first switched us from a 403 to a 401, I feel confident saying it's not the default.

I _think_ the default might have been ```
AUTHENTICATION_BACKENDS = [
    'rules.permissions.ObjectPermissionBackend',
    'openedx.core.djangoapps.oauth_dispatch.dot_overrides.backends.EdxRateLimitedAllowAllUsersModelBackend',
    'bridgekeeper.backends.RulePermissionBackend',
]
```
based  on the lms/envs/common.py file.